_posts/2020-07-16-omero-5-6-2.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.2 and OMERO.web 5.7.0
+intro-blurb: This release includes OMERO.server 5.6.2, OMERO.web 5.7.0 and OMERO.py 5.7.1
+---
+
+Today we are releasing OMERO.server 5.6.2 which fixes installaion instructions for CentOS 7, 
+adds installation instructions for both OMERO.web and OMERO.server on CentOS 8 and Ubuntu 20.04,
+and finally removes installation instructions on Ubuntu 16.04. 
+We are also releasing OMERO.web 5.7.0 and OMERO.py 5.7.1 which both contain some API additions.
+
+OMERO.server 5.6.2 runs in the same deployment environments as OMERO.server 5.6.0 and
+the Bio-Formats Memoizer cache will be invalidated.
+
+### Installing the Software:
+
+For full details of the changes with the OMERO 5.6 series see the
+[OMERO 5.6.0]({{ site.baseurl }}/2020/01/15/omero-5-6-0.html) release
+announcement. OMERO.server 5.6.1 is designed to be a small step beyond the 5.6.0 release
+in order to simplify upgrade. Full documentation for this release is available
+under <https://docs.openmicroscopy.org/omero/5.6.1/>.
+
+OMERO.server 5.6.2 is available from
+[archived downloads](https://downloads.openmicroscopy.org/omero/5.6.2/artifacts/)
+and [omero-web 5.7.0](https://pypi.org/project/omero-web/5.7.0/).
+These have been tested with
+[omero-py 5.7.1](https://pypi.org/project/omero-py/5.7.1/) so we
+recommend that you upgrade OMERO.py on both server and web deployments.
```

Comment:
For OMERO.web, is `omero-py` going to be automatically upgraded?

Reply:
only if you run ``--upgrade``

Reply:
Even with https://github.com/ome/omero-web/commit/c2c419865f0afaf84f84c712d2b46c11ccd8c3d4 ?

Reply:
it should be updated. i forgot about that condition